src/main/java/com/stripe/param/InvoiceUpcomingParams.java

```diff
@@ -1046,6 +1076,392 @@ public Builder setStart(Long start) {
         }
       }
     }
+
+    @Getter
+    public static class PriceData {
```

Comment:
Is it safe to pass `price_data` to an upcoming endpoint? Do we create a fake price/product under the hood?